fennel/client_tests/test_incremental.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+
+import pytest
+
+from fennel.connectors import Webhook, source
+from fennel.datasets import dataset, field
+from fennel.testing import mock
+
+__owner__ = "nitin@fennel.ai"
+webhook = Webhook(name="fennel_webhook")
+
+
+@pytest.mark.integration
+@mock
+def test_dataset_version_change(client):
```

Comment:
can you modify the test to change something and actually assert the change took place.
for eg you had a pipeline with transform doing x2 and later x3, on version increment with incremental mode we should see x3

Reply:
changed